### PR TITLE
add HARC-HAKRCF405D.config

### PR DESCRIPTION
**Pull request by WilsonChang**
Hi,
please review and add HARC-HAKRCF405D unified configure target file for HAKRC.
thanks.

### CODE REVIEW

File: configs/default/HARC-HAKRCF405D.config [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+# serial
+serial 1 64 115200 57600 0 115200
+
+# master
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set mag_hardware = NONE
```
Comment (mikeller) [inline configs/default/HARC-HAKRCF405D.config:121]: If this is set, this means that mag devices will not be supported on this flight controller, and users asking for instructions will be told to buy a different flight controller.
If you do not want this, please remove this line - the default setting works just fine when no mag device is connected.

File: configs/default/HARC-HAKRCF405D.config [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+set serialrx_provider = SBUS
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set motor_pwm_protocol = DSHOT600
```
Comment (mikeller) [inline configs/default/HARC-HAKRCF405D.config:125]: Is this target only going to be used for an all-in-one board with integrated ESCs that support Dshot600?

**Review by mikeller — CHANGES_REQUESTED** [review]

File: configs/default/HARC-HAKRCF405D.config (hunk 1 above)
Comment (WilsonChang) [reply configs/default/HARC-HAKRCF405D.config:121]: OK!

File: configs/default/HARC-HAKRCF405D.config (hunk 2 above)
Comment (WilsonChang) [reply configs/default/HARC-HAKRCF405D.config:125]: Only a suggestion. OK I delete this.

**Review by mikeller — APPROVED** [review]

**Pull request merged**